### PR TITLE
Toned down pull request

### PR DESCRIPTION
**Pull request by dskvr**
Removed unstable effects. Includes angular changes. Server-simple.js for client testing.

### CODE REVIEW

**Comment by mattolson** [comment]
Thanks, Sean. I think we're a lot closer. Can you please remove those DS_Store files, and also change the file mode back for anything that changed. There's no need for any of these files to be executable. Look at the "files changed" tab in this pull request and you'll see where they are. In addition, I'll make some comments inline.

File: driver/effects/fade_to.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+var path = require('path');
+var util = require('util');
+var Effect = require('../effect');
+
+var NAME = path.basename(__filename, '.js'); // Our unique name
+
+// This effect simply sets the entire grid to a single color
```
Comment (mattolson) [inline driver/effects/fade_to.js:8]: Can you update the documentation for this, and the other new effects, both to describe it and it's options.

File: driver/effects/fade_to.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	this.current_step++;
+
+	if (this.current_step >= this.number_of_steps) {
+		this.current_step = 0; 
+		if(this.random_color) this.color = this.randomColor();
+	}
+  return true;
+};
+
+FadeTo.prototype.randomColor = function(){
```
Comment (mattolson) [inline driver/effects/fade_to.js]: Please move this to color_utils, seems useful for everyone.

File: driver/effects/fade_to.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	return [ Math.floor( Math.random() * 255 ), Math.floor( Math.random() * 255 ), Math.floor( Math.random() * 255 ) ] ;
+}
+
+// Return js object containing all params and their types
+FadeTo.options = function() {
```
Comment (mattolson) [inline driver/effects/fade_to.js:83]: Please update this to include the actual options for this effect. Same thing for all new effects.

File: driver/effects/light_bulb.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+		idex = 0, 
+		idex_offset = 0, 
+		ihue = 0, 
+		isat =0, 
+		isat = 0, 
+		tcount = 0.0, 
+		lcount = 0;
+
+function LightBulb(grid, options)
+{kaz
```
Comment (mattolson) [inline driver/effects/light_bulb.js]: kaz?

File: driver/effects/light_bulb.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+var path = require('path');
+var util = require('util');
+var Effect = require('../effect');
+
+var NAME = path.basename(__filename, '.js'); // Our unique name
+
+var STEPS = 100;
+
+var bouncedirection = 0,
```
Comment (mattolson) [inline driver/effects/light_bulb.js]: Should these be member variables?

File: driver/grid.js [hunk 6]
```diff
@@ -159,5 +220,62 @@ Grid.prototype.addListener = function(listener) {
   }
 };
 
+Grid.prototype.HSVtoRGB = function(h, s, v){
```
Comment (mattolson) [inline driver/grid.js]: This already exists in color_utils

File: driver/static/js/services.js [hunk 7]
```diff
@@ -1,30 +1,95 @@
 'use strict';
 
+blink
```
Comment (mattolson) [inline driver/static/js/services.js:3]: blink?

Comment (dskvr) [inline driver/static/js/services.js:3]: Haha, can of worms ;) ... let's get that through and I'll take blink out of the global scope in next request. Promise. I'll buy you a beer.

**Comment by dskvr** [comment]
Pretty sure that about sums it up. Let's get this rolling. Would love option setter methods!

**Pull request merged**